tests/prediction/time_series/pipelines/transfer_learning_pipeline_test.py

```diff
@@ -38,3 +41,20 @@ def test_transfer_learning_pipeline_results_correct_type_and_format_3_spec() ->
     assert results.shape == (3, 2)
     assert isinstance(results["Keras Dense Neural Network"], pd.Series)
     assert isinstance(results["PyTorch Lightning Dense Neural Network"], pd.Series)
+
+
+def test_transfer_learning_pipeline_export() -> None:
+    EXPORT_PATH = "tests/prediction/time_series/test_data/export"
```

Comment:
See above.

Reply:
Changed accordingly. Please look whether this comment is still relevant.